scripts/deps.py

```diff
@@ -1,59 +1,106 @@
 import os
-from os.path import abspath, dirname, join
 import platform
 import sys
-
 import jinja2
 import setuptools
 import yaml
+from unittest import mock
+from functools import reduce
+from operator import add
+from pathlib import Path
```

Comment:
This is available only on Python 3. I think instead of making this compatible with Python 2, this branch could be rebased on `landing-2.0`, which is 3 only (especially we will be merging it soon).

Reply:
@mattpap I rebased this branch on `landing-2.0`.